tests/document_ops.rs

```diff
@@ -504,267 +518,291 @@ fn doc_create_must_grant() -> Result<(), IronOxideErr> {
+                true,
+                vec![],
+            ),
+        )
+        .await?;
     let doc_id = doc_result.id().clone();
 
     // create a second user to grant/revoke access to the document
-    let user = create_second_user();
+    let user = create_second_user().await;
 
-    let group_result = sdk.group_create(&Default::default())?;
+    let group_result = sdk.group_create(&Default::default()).await?;
     let group_id = group_result.id().clone();
 
-    let grants = sdk.document_grant_access(
-        &doc_id,
-        &vec![
-            UserOrGroup::User {
-                id: user.account_id().clone(),
-            },
-            UserOrGroup::Group {
-                id: group_id.clone(),
-            },
-        ],
-    )?;
+    let grants = sdk
+        .document_grant_access(
+            &doc_id,
+            &vec![
+                UserOrGroup::User {
+                    id: user.account_id().clone(),
+                },
+                UserOrGroup::Group {
+                    id: group_id.clone(),
+                },
+            ],
+        )
+        .await?;
     assert_eq!(grants.succeeded().len(), 2);
 
-    let revokes = sdk.document_revoke_access(
-        &doc_id,
-        &vec![
-            UserOrGroup::User {
-                id: user.account_id().clone(),
-            },
-            UserOrGroup::Group {
-                id: group_id.clone(),
-            },
-            UserOrGroup::User {
-                id: "bad-user-id".try_into()?,
-            },
-            UserOrGroup::Group {
-                id: "bad-group-id".try_into()?,
-            },
-        ],
-    )?;
+    let revokes = sdk
+        .document_revoke_access(
+            &doc_id,
+            &vec![
+                UserOrGroup::User {
+                    id: user.account_id().clone(),
+                },
+                UserOrGroup::Group {
+                    id: group_id.clone(),
+                },
+                UserOrGroup::User {
+                    id: "bad-user-id".try_into()?,
+                },
+                UserOrGroup::Group {
+                    id: "bad-group-id".try_into()?,
+                },
+            ],
+        )
+        .await?;
 
     assert_eq!(revokes.succeeded().len(), 2);
     assert_eq!(revokes.failed().len(), 2);
     Ok(())
 }
 
-#[test]
-fn doc_encrypt_concurrent() -> Result<(), IronOxideErr> {
-    let sdk = Arc::new(initialize_sdk()?);
-    let doc = [43u8; 64];
-    let _encrypted_doc = sdk.document_encrypt(&doc, &Default::default())?;
-
-    let mut threads = vec![];
-    for _i in 0..10 {
-        let sdk_ref = sdk.clone();
-        threads.push(std::thread::spawn(move || {
-            let _result = sdk_ref.document_encrypt(&doc, &Default::default()).unwrap();
-        }));
-    }
-
-    let mut joined_count = 0;
-    for t in threads {
-        t.join().expect("couldn't join");
-        joined_count += 1;
-    }
-
-    assert_eq!(joined_count, 10);
-    Ok(())
-}
+//#[tokio::test]
```

Comment:
Since we have throughput test/benches coming, I think I'm going to remove this test completely. Comment if you disagree.

Reply:
Feel free.